Police-Data-Accessibility-Project/data-sources-app-v2
# Mc 508 create user user id get endpoint

### Fixes

* https://github.com/Police-Data-Accessibility-Project/data-sources-app/issues/508

### Description

* Create `user/{user_id}` `GET` endpoint, and associated logic, tests, and documentation
* Refactor test generation and management logic.

### Testing

* Run tests in `tests` directory to confirm functionality 
* Inspect API to confirm expected content in `user/{user_id}` `GET` 

### Performance

* Test overhead increased

### Docs

* API documentation added

### Breaking Changes

* No breaking changes.
